components/logins/src/lib.rs

```diff
@@ -16,6 +16,9 @@ mod update_plan;
 mod util;
 
 mod ffi;
+pub mod msg_types {
+    include!(concat!(env!("OUT_DIR"), "/msg_types.rs"));
```

Comment:
From what I've read in the tests, the environment variable `OUT_DIR` is not set. Maybe this helps ;)

Reply:
Thanks for the tip @schoenenberg, I think you're right; this is probably due to missing a `build.rs` for the `logins` component. Adding a `build.rs` based on the one in e.g. `components/places/build.rs` (and the file `components/places/src/places_msg_types.proto` that it uses) looks like the next step here.

Reply:
@tinoism Are you doing this?

Reply:
Or please add give me the permission to push on tinoism:bug_1605

Reply:
@schoenenberg feel free to fork this branch and open a new pull request 👍